extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/binder/CommonQueryBinder.java

```diff
@@ -6,8 +6,7 @@
 import java.time.LocalDateTime;
 import java.time.ZoneOffset;
 import java.time.format.DateTimeFormatter;
-import java.util.Date;
-import java.util.TimeZone;
+import java.util.*;
```

Comment:
Wildcard imports are prohibited

Reply:
Done.

Reply:
I don't know why IntelliJ adds replace this for all the package, I'll check later.